### ISSUE TITLE
Load a different set of settings

### ISSUE THREAD

**Issue by Darkmaster006** [irzyxa/Volume2#283]
Hello! First of all, I want to thank you for this awesome program, it's really really amazing to be able to control volume and other things with such ease. As for my suggestion, what I thought would be nice to have: a place where we could change (maybe with some sort of config file that can be loaded?) between different "profiles". For example, I have two different keyboard with two different set of keys, so that some shortcuts will differ (many, actually), is there a way to "load a profile" for each with my already configured shortcuts and other stuff? Would be a nice feature to have. Thank you.

**Comment by irzyxa** [comment]
Yes, it can be done by running program with different parameters (for example create 2 shortcuts):
_Volume2.exe /cfn d:\Projects\Volume2\Config_keyboard1.ini
Volume2.exe /cfn d:\Projects\Volume2\Config_keyboard2.ini_
![image](https://github.com/irzyxa/Volume2/assets/5119360/76d16e0b-8bc1-413d-8472-82eaf5839d95)
